### PR TITLE
complete boilerplate file

### PR DESCRIPTION
**Pull request by zouyee**
Fixes #

## Proposed Changes

* complete boilerplate file 

**Release Note**

```
NONE

```

### CODE REVIEW

File: hack/boilerplate/boilerplate.sh.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Knative Authors
```
Comment (vdemeester) [inline hack/boilerplate/boilerplate.sh.txt]: Shouldn't it be `2018-2019`

**Comment by adrcunha** [comment]
> This seems like something that should be shared in knative/pkg (or knative/test-infra?) and vendored into this repo, since presumably it will be used by all the Knative repos. I don't want Build to be the only repo that enforces/automates this while the other repos do something else.

This is a copy of the script in serving. It's a convenience script, but I'm not sure how much usage it gets. If you think it's just adding unnecessary stuff to build, feel free to reject it.

/uncc @adrcunha

**Pull request closed without merging**